modules/ocl-loader/src/main/java/org/eclipse/xpanse/modules/ocl/loader/data/models/enums/DeployerKind.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.ocl.loader.data.models.enums;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/**
+ * Deployer kinds.
+ */
+public enum DeployerKind {
+    TERRAFORM("terraform"),
+    PULUMI("pulumi");

```

Comment:
i think we can remove this 2nd value for now as we do not support it for now. we examples in the field description if needed.